scripts/init-project.sh

```
@@ -32,14 +32,13 @@ fi
 if [ -z "$PROJECT_TYPE" ]; then
     echo "Project types:"
     echo "  1) screenplay"
-    echo "  2) novel"
-    echo "  3) film-production"
-    read -p "Select project type [1-3]: " TYPE_NUM
+    echo ""
+    echo -e "${YELLOW}  novel and film-production are planned, not shipped. The harness has no${NC}"
+    echo -e "${YELLOW}  commands for them yet, so initializing one leaves you with no tooling.${NC}"
+    read -p "Select project type [1]: " TYPE_NUM
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)